docs/dev/ci/github_actions/runners.md

```diff
@@ -38,7 +38,14 @@ OpenVINO repositories make use of the following runners:
 
 ## Available Self-hosted Runners
 
-The self-hosted runners are dynamically spawned for each requested pipeline.
+As of 25/05/2024, two groups of the self-hosted runners are available:
```

Comment:
The date/time format is different in UK/Ireland vs US, so I would write the month instead here and in other similar places.
But I think "currently" or "at the moment" is also OK assuming we will update the documentation when something changes.